src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -66,34 +64,11 @@ protected void textChanged(final DocumentEvent e) {
       }
     });
 
-    final TextComponentAccessor<JComboBox> textComponentAccessor = new TextComponentAccessor<JComboBox>() {
-      @Override
-      public String getText(final JComboBox component) {
-        return component.getEditor().getItem().toString();
-      }
-
-      @Override
-      public void setText(@NotNull final JComboBox component, @NotNull final String text) {
-        if (!text.isEmpty() && !FlutterSdkUtil.isFlutterSdkHome(text)) {
```

Comment:
This logic was written for the following use case:
- assume I have `dart` folder with `dart-sdk` and `dartium` subfolders
- in file chooser I select not the `dart-sdk` folder, but its parent - `dart` folder
- ide notices that selected path is not a dart sdk folder and looks for dart-sdk subfolder
- if dart-sdk subfolder contains Dart SDK then IDE behaves as if user has chosen correct folder himself

This logic is not applicable for Dart because it is unlikely that Flutter SDK is in `dart-sdk` folder and user selects parent folder instead.